packages/manager-components/src/components/content/headers/headers.component.tsx

```diff
@@ -16,13 +15,13 @@ export const Headers: React.FC<HeadersProps> = ({
   headerButton,
 }) => {
   return (
-    <div className="pl-5 flex items-center justify-between">
+    <div className="flex items-center justify-between">
       <div>
-        {title && <Title>{title}</Title>}
+        {title && <Title className="block mb-8">{title}</Title>}
```

Comment:
Personally I am not a big fan of including margin inside typography component
It is easier to import 0 margin components and manage all the margins in the templates in my opinion.

And by the way, it will break pci-rancher design when merging the PR if I change the default margin of the component.

Reply:
we can let title component without classes and add a class for title component in headers if you are sure the templates will be use <headers> component.

Title can stay with 'className' attributes to be less restictive as we didn't know all cases in the future templates

Reply:
A general rule of thumb is to avoid hard coded margin on components, it's the responsibility of the parent component to specify spacing between its children.